### DISCUSSION TITLE
Misleading Doc statement about Fastlane folder structure

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#227] [Q&A]
I remember reading this statement in the Docs a few weeks back...  it was mis-interpreted by ME to mean the structures would differ... and I needed to start over again.  I was WRONG.  Not the first nor the last time.

**"The structure of the metadata for each platform is different. For iOS and other Apple platforms, the Darwin/ folder will look like this:"**

in Doc: https://skip.tools/docs/deployment/#fastlane

<img width="1067" alt="Screenshot 2024-09-08 at 11 53 23 AM" src="https://github.com/user-attachments/assets/0b933ad2-700f-41a1-9f31-dddb72b9e729">

Maybe an **English Major** could make it better...

**Comment by marcprux** [comment]
The folder structures for Android and iOS are indeed different, and the metadata keys are also not consistent between each other. The documentation statement is meant to convey that there is not one single shared Fastlane configuration for you project, but rather than each target with have its own separate configuration.

**Comment by davidakoontz** [comment]
So as I've learned a lot more... and not as ignorant as I was when I read that and mistakenly thought I needed to start completely fresh.

The Skip structure of folders is the SAME.  <<== important distinction 

The Fastlane folders differ (inside the Skip Structure).

I believe one could start a project with Skip (without Fastlane) and then later add Fastlane (with fastlane init ) and ALL would be OK.

**Reply by marcprux** [reply]
> I believe one could start a project with Skip (without Fastlane) and then later add Fastlane (with fastlane init ) and ALL would be OK.

As of recently, `skip init` will default to creating a fastlane configuration (i.e., the "--fastlane" argument now defaults to true). It may be that your project was created before we enabled this by default.

But you are right that it would be nice to be able to add a fastlane configuration to an existing project that doesn't already have it.
